packages/api/.env.sample

```
@@ -6,4 +6,8 @@ PORT="3000"
 DATABASE_URL="postgresql://localhost:5432/project-x"
 
 # Database user is optional
-# DATABASE_USER=""
\ No newline at end of file
+# DATABASE_USER=""
+
+## Github keys ##
+GITHUB_ID="key"
```

If this is the client id, this should probably match the full name

```suggestion
GITHUB_CLIENT_ID="key"
```